deis/monitor
# fix(grafana): Fix blank dashboards from appearing in dropdown

Move the dashboards from /usr/share/grafana/dashboards which is where we scan for other json dashboards to /usr/share/grafana/api/dashboards.

Resolves this issue - https://github.com/grafana/grafana/issues/5569

Manual test steps:
1. Checkout PR
2. Use `make build push upgrade` to deploy new grafana image
3. Log into `grafana.mydomain.com` `admin/admin`
4. Verify you do not see any blank dashboards in dropdown.
